src/js/models/render-node.js

```diff
@@ -1,75 +1,47 @@
-export default class RenderNode {
+import LinkedItem from "content-kit-editor/utils/linked-item";
+import LinkedList from "content-kit-editor/utils/linked-list";
+
+export default class RenderNode extends LinkedItem {
```

Comment:
I would prefer composition over inheritance here — we have a function in utils called `mixin` that can be used instead. This is fine for now, though.